### PR TITLE
Docs: Review documentation

### PR DESCRIPTION
**Pull request by 0xGabi**
Review the documentation updating a few places.

### CODE REVIEW

File: docs/advanced/app-connectors.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ The connectors are composed of a root project which implements a base connector
 
 ### The connector
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:11]: Should we add as an intro that we are describing the structure that we use in our own connectors (and that we recommend to follow)?

File: docs/advanced/app-connectors.md [hunk 2]
```diff
@@ -10,13 +10,16 @@ The connectors are composed of a root project which implements a base connector
...
 
-The `src` folder contains a connector implementation for The Graph which uses `@aragon/connect-thegraph` to perform GraphQL queries, parse them, and present the result in the form of entity objects compatible with the `@aragon/connect` API.
+The `src` folder contains:
 
-Use `index.ts` to pick which entities and objects are exposed to other packages.
+- `models` folder: objects compatible with the `@aragon/connect` API.
+- `thegraph` folder: a connector implementation for The Graph which uses `@aragon/connect-thegraph` to perform GraphQL queries, parse them and present the result in the form of models objects. The `connector.ts` that has the app functions.
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:16]:
```suggestion
- `thegraph` folder: a connector implementation for The Graph which uses `@aragon/connect-thegraph` to perform GraphQL queries, parse them and present the result in the form of models objects. This part is only used internally, and `connector.ts` contains the methods responsible to fetch the app data from The Graph.
```

File: docs/advanced/app-connectors.md [hunk 3]
```diff
@@ -10,13 +10,16 @@ The connectors are composed of a root project which implements a base connector
 
 ### The connector
 
-The `src` folder contains a connector implementation for The Graph which uses `@aragon/connect-thegraph` to perform GraphQL queries, parse them, and present the result in the form of entity objects compatible with the `@aragon/connect` API.
+The `src` folder contains:
 
-Use `index.ts` to pick which entities and objects are exposed to other packages.
+- `models` folder: objects compatible with the `@aragon/connect` API.
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:15]:
```suggestion
- `models` folder: objects compatible with the `@aragon/connect` API. This is what the consumers of your app connector are going to interact with.
```

File: docs/advanced/app-connectors.md [hunk 4]
```diff
@@ -10,13 +10,16 @@ The connectors are composed of a root project which implements a base connector
...
+- `thegraph` folder: a connector implementation for The Graph which uses `@aragon/connect-thegraph` to perform GraphQL queries, parse them and present the result in the form of models objects. The `connector.ts` that has the app functions.
 
-In `connector.ts`, add functions that the entities of your connector will use. For example, the `Vote` entity will call its `castsForVote(...): Promise<Cast[]>` function. These functions all follow the same structure of \(1\) performing a query, \(2\) parsing the results of a query, and \(3\) wrapping and returning the results in the appropriate entity.
+Use `index.ts` to pick which models and objects are exposed to other packages.
 
-Each of the steps described above is separated in the `entities`, `parsers`, and `queries` folders for clarity.
+Use `connect.ts` to create the connector logic using `createAppConnector()` from `'@aragon/connect-core'`.
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:20]:
```suggestion
Use `connect.ts` to create the connector logic [using `createAppConnector()`](#create-the-connector) from `'@aragon/connect-core'`.
```

File: docs/advanced/app-connectors.md [hunk 5]
```diff
@@ -10,13 +10,16 @@ The connectors are composed of a root project which implements a base connector
...
+
+In `connector.ts`, add functions that the models of your connector will use. For example, the `Vote` model will call its `castsForVote(...): Promise<Cast[]>` function. These functions all follow the same structure of \(1\) performing a query, \(2\) parsing the results of a query, and \(3\) wrapping and returning the results in the appropriate model.
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:22]:
```suggestion
In `thegraph/connector.ts` (assuming your connector is using The Graph), add functions that the models of your connector will use. For example, the `Vote` model of `@aragon/connect-voting` [will call its `castsForVote(...): Promise<Cast[]>` function](https://github.com/aragon/connect/blob/12dec7e5147220d29fb960bff01ff95e9ccca1bf/packages/connect-voting/src/models/Vote.ts#L39). These functions all follow the same structure of \(1\) performing a query, \(2\) parsing the results of a query, and \(3\) wrapping and returning the results in the appropriate model.
```

File: docs/advanced/app-connectors.md [hunk 6]
```diff
@@ -44,7 +47,7 @@ export default createAppConnector(() => ({
 
 From the point of view of a consumer of your connector, the resulting object will be a function that can get used to connect an app.
 
-For example, this is how app authors can initiate a connection to the Voting app. In this example, `connectVoting` got returned by `createAppConnector()`:
+For example, this is how app authors can initiate a connection to the Voting app. In this example, `connectVoting` got returned by the implementation of `createAppConnector()` of the Voting connector:
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:50]: I’m not sure I understand this, the implementation of `createAppConnector()` is not in the `Voting` connector? :thinking:

File: docs/advanced/app-connectors.md [hunk 7]
```diff
@@ -102,7 +105,7 @@ Here are the parameters passed to the `createAppConnector()` callback:
 
 #### appConnect\(app, connector\)
 
-The function returned by `createAppConnector()`, called by app authors, takes these parameters:
+The function returned by `createAppConnector()`, called by app authors. Takes these parameters:
```
Comment (bpierre) [inline docs/advanced/app-connectors.md:108]:
```suggestion
The function returned by `createAppConnector()`, called by app authors. It takes these parameters:
```

File: docs/guides/getting-started.md [hunk 8]
```diff
@@ -168,11 +170,10 @@ Generating a transaction is done in three steps. First, call a method returning
 const intent = await org.appIntent(voting, 'vote', [votes[0].id, true, true])
 ```
 
-Then to retrieve the path you want, pass the account that will be signing the transaction. Aragon Connect will go through the permissions of the organization, and return all possible paths:
+Then to retrieve the path you want, pass the account that will be signing the transaction. Aragon Connect will go through the permissions of the organization, and return the shortest path:
 
 ```javascript
-// The first path is the shortest
-const [path] = intent.paths(wallet.account)
+const path = intent.paths(wallet.account)
```
Comment (bpierre) [inline docs/guides/getting-started.md:176]: I can’t wait for the new API, returning a single value on a method named as a plural feels really weird. I was wondering if in the meantime, we should add an alias on `intent#path()` and only refer to `.path()` in the documentation?

Comment (bpierre) [reply docs/guides/getting-started.md:176]: After having thought more about it: let’s not introduce any change and just move to the new API.

File: docs/advanced/app-connectors.md (hunk 6 above)
Comment (0xGabi) [reply docs/advanced/app-connectors.md:50]: You are right, I tried to clarify and I complicate it more 😅

File: docs/guides/getting-started.md (hunk 8 above)
Comment (0xGabi) [reply docs/guides/getting-started.md:176]: As discussed offline we decide to not change the API given a major change will be introduced soon.

**Pull request merged**